src/controller/user-controller.ts

```diff
@@ -1627,7 +1631,19 @@ export default class UserController extends BaseController {
         return;
       }
 
-      await DebtorService.waiveFines(id);
+      const totalAmountOfFines = user.currentFines!.fines.reduce((total, f) => total.add(f.amount), Dinero());
+      // Backwards compatibility with old version, where you could only waive all user's fines
```

Comment:
Related to my previous remark, why not split it into two routes?

Reply:
Because you can still waive all fines with this endpoint, but you have to give the total amount of fines manually. Making two endpoints will in my opinion just be confusing, as both can do the same thing. When do you use which endpoint?